examples/trader_example.py

```diff
@@ -0,0 +1,67 @@
+"""
+Cinyoung Hur, cinyoung.hur@gmail.com
+James Park, laplacian.k@gmail.com
+seoulai.com
+2018
+"""
+import seoulai_gym as gym
+from seoulai_gym.envs.traders.agents import RandomAgentBuffett
+
+
+def main():
```

Comment:
Example code does not seem to work well for me (on MacOS, I will try Linux later) or I am not sure what it should do :)

Reply:
There was a matplotlib issue. James is working on this.